src/ast.scm

```diff
@@ -329,8 +329,8 @@
         (else
          (case (car v)
            ((...)
-	    (arg-name (cadr v)) ;; to check for errors
-	    (decl-var (cadr v)))
+	        (arg-name (cadr v)) ;; to check for errors
+	        (decl-var (cadr v)))
```

Comment:
I think the real problem is that there are tab characters on this line, and the other lines use spaces. So, just convert the tabs to spaces instead.

Reply:
TIL `M-x untabify`